enterprise/server/usage/usage.go

```diff
@@ -254,12 +264,9 @@ func (ut *tracker) flushCounts(ctx context.Context, groupID string, c collection
 		PeriodStartUsec: timeutil.ToUsec(c.UsagePeriod().Start()),
 	}
 	dbh := ut.env.GetDBHandle()
-	return dbh.Transaction(ctx, func(tx *db.DB) error {
```

Comment:
https://www.percona.com/blog/2014/10/28/how-to-deal-with-mysql-deadlocks/

Reply:
(Working on this)

Reply:
Decided that we'll keep this in a transaction and that we can tolerate these deadlocks once in a blue moon; the worst that happens is that we just log a warning -- MySQL will ensure that at least one of the contending transactions will go through.

Reply:
SG